src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -739,7 +754,7 @@ const formConfig = {
           path: 'fully-developed-claim',
           uiSchema: fullyDevelopedClaim.uiSchema,
           schema: fullyDevelopedClaim.schema,
-          depends: formData => !isBDD(formData),
+          depends: formData => !isBDD(formData) && environment.isProduction(),
```

Comment:
would `isProduction()` make this untestable in staging?

Reply:
Team had a quick huddle to go over this story. Yes, that is the goal to hide this view on staging so that we can go through the collab cycle's staging review before releasing to prod. There is another subsequent story to clean this up once we have approval for launch. We also discussed toggling challenges and collab cycle process